SRC/ReplaceMultiplicationWithMinus.cpp

```diff
@@ -0,0 +1,46 @@
+#include "ReplaceMultiplicationWithMinus.h"
+ReplaceMultiplicationWithMinus* ReplaceMultiplicationWithMinus::m_Instance = nullptr;
+
+static constexpr char PLUS = '+';
+static constexpr char MINUS = '-';
+static constexpr char MULTIPLICATION = '*';
+static constexpr char DIVISION = '/';
+static constexpr char GREATER = '>';
+static constexpr char DOT = '.';
+static constexpr char WHITESPACE = ' ';
+static constexpr char SEMICOLON = ';';
+
+ReplaceMultiplicationWithMinus::ReplaceMultiplicationWithMinus()
```

Comment:
We can either replace multiplication with + or - why do we have both the classes ?